### DISCUSSION TITLE
Question about calculating dA/dt

### DISCUSSION THREAD

**Discussion by windyjunfeng** [simnibs/simnibs#363] [8. Simulation workflows]
Hi, I have a special question. It's known that the calculation of electric field involves two part, i.e., the gradient of potential, and the derivative of magnetic vector potential (dA/dt). As for dA/dt, can I calculate it only once for one subject when optimizing the coil positions? Actually, dA/dt doesn't depend on the tissue conductivity in the head. I have modified the code to calculate dA/dt in a large space under the coil. When calculating the electric field with specific stimulation, I mask the dA/dt calculated before by the head model in coil space and get the final dA/dt. But I find it's different from dA/dt normally calculated by SimNIBS. I think it's due to the different representation of nii.gz (my output) and .msh. 
Of course, the calculation of dA/dt is quick in the current version. I have a special purpose and wonder whether my thought is right or not. Thank you~

**Comment by bjornsig** [comment]
Because of the different representations of the coils, numerical differences in the calculated da/dt are expected but should be small enough to be irrelevant in practice. You can compare your file to a file created with:
https://simnibs.github.io/simnibs/build/html/documentation/command_line/ccd2nifti.html#ccd2nifti
and see if they are different.

You can calculate the nifti file once per coil. You do not need to do it per subject.
